plugins/catalog-backend/src/search/DefaultCatalogCollatorFactory.test.ts

```diff
@@ -150,6 +160,36 @@ describe('DefaultCatalogCollatorFactory', () => {
       });
     });
 
+    it('maps a returned entity to an expected CatalogEntityDocument with custom transformer', async () => {
```

Comment:
It doesn't look like this is actually testing what it says it's testing.  Could you update?

Reply:
oh, good catch, thanks